core/src/test/java/com/novoda/noplayer/internal/exoplayer/ExoPlayerFacadeTest.java

```diff
@@ -206,6 +208,20 @@ public void selectSubtitleTrack_thenThrowsIllegalStateException() {
 
             facade.selectSubtitleTrack(subtitleTrack);
         }
+
+        @Test
+        public void whenSetVolume_thenThrowsIllegalStateException() {
+            thrown.expect(ExceptionMatcher.matches("Video must be loaded before trying to interact with the player", IllegalStateException.class));
+
+            facade.setVolume(ANY_VOLUME);
+        }
+
+        @Test
+        public void whenGetVolume_thenthenThrowsIllegalStateException() {
```

Comment:
extra `then`

Reply:
I'd avoid the `withoutLoadingVideoFirst` because the enclosing class is named `GivenVideoNotLoaded`

Reply:
26a688c